corporate/tests/test_stripe.py

```diff
@@ -2373,10 +2373,22 @@ def test_update_billing_method_of_current_plan(self) -> None:
         update_billing_method_of_current_plan(realm, True)
         plan.refresh_from_db()
         self.assertEqual(plan.charge_automatically, True)
+        realm_audit_log = RealmAuditLog.objects.filter(
+            event_type=RealmAuditLog.REALM_BILLING_METHOD_CHANGED
+        ).order_by("-id")[0]
+        expected_extra_data = {"charge_automatically": plan.charge_automatically}
+        self.assertEqual(realm_audit_log.acting_user, None)
+        self.assertEqual(realm_audit_log.extra_data, str(expected_extra_data))
```

Comment:
Pass `acting_user` in one of the changes and check whether it's logged correctly.